src/desktop/apps/auction_reaction/routes.tsx

```diff
@@ -0,0 +1,63 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import { routes } from "reaction/Apps/Auction/routes"
+import { stitch } from "@artsy/stitch"
+
+export const bidderRegistration = async (req, res, next) => {
+  if (!res.locals.sd.CURRENT_USER) {
+    return res.redirect(
+      `/login?redirectTo=${encodeURIComponent(req.originalUrl)}`
+    )
+  }
+  try {
+    const {
+      bodyHTML,
+      redirect,
+      status,
+      headTags,
+      scripts,
+      styleTags,
+    } = await buildServerApp({
+      routes,
+      url: req.url,
+      userAgent: req.header("User-Agent"),
+      context: buildServerAppContext(req, res),
+    })
+
+    if (redirect) {
```

Comment:
What's the purpose of this code? If there is a redirect prop passed into the component, then redirect the component and don't render the registration form? What's the use-case for this?